test-packages/e2e-tests/test/request-builder.spec.ts

```diff
@@ -1,11 +1,48 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import { TestEntity } from '@sap-cloud-sdk/test-services-e2e/v4/admin-service';
+import moment from 'moment';
 
 const url = 'http://localhost:4004/';
 const destination = { url };
+interface EntityKey {
+  keyString: string;
+  keyInt: number;
+}
+const entityKey = { keyString: 'keyString', keyInt: 123 };
+
+async function queryEntity(key: EntityKey): Promise<TestEntity> {
+  return TestEntity.requestBuilder()
+    .getByKey(key.keyInt, key.keyString)
+    .execute(destination);
+}
+
+async function deleteEntity(key: EntityKey): Promise<void> {
+  try {
+    const fetched = await queryEntity(key);
+    return TestEntity.requestBuilder().delete(fetched).execute(destination);
+  } catch (e) {
+    // Entity was not present nothing to delete
```

Comment:
maybe check the error message and only ignore when it's about "nothing to delete"